components/hab/static/template_init

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
```

Comment:
This generated file doesn't contian sensible defaults, either. Unless what we are generating is useful out of the box or provides some commented out instructions let's just not generate them.

I'm not sure that we can provide sensible defaults for an init/run hook